src/hooks/useUSDPrice.ts

```diff
@@ -41,8 +41,8 @@ function useETHValue(currencyAmount?: CurrencyAmount<Currency>): {
     }
```

Comment:
Why aren't this function's return values memoized?

Reply:
i feel like these types of arguments should be the other way around since the default is _not_ memoized - like there needs to be a reason for memoizing vs the other. in this case the consumers destructure the props, so memoizing is unnecessary for maintaining referential stability